service/controller/v1/resource/drainer/create.go

```diff
@@ -30,10 +30,12 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		return microerror.Mask(err)
 	}
 
+	clusterID := key.ClusterIDFromDrainerConfig(drainerConfig)
```

Comment:
You could use LoggerMeta: https://github.com/giantswarm/micrologger/blob/master/loggermeta/logger_meta.go
So you would set this value once here and we'd have it everywhere.